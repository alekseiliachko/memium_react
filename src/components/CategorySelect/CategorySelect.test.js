import React from "react";
import renderer from "react-test-renderer";
import { CategorySelect } from "./index";

describe("category select", () => {
  it("all categories render correctly", async () => {
    let tree;

    renderer.act(() => {
      tree = renderer.create(<CategorySelect tab={0} />);
    });

    expect(tree.toJSON()).toMatchSnapshot();

    for (let i = 1; i < 6; i++) {
      renderer.act(() => {
        tree.update(<CategorySelect tab={i} />);
      });

      expect(tree.toJSON()).toMatchSnapshot();
    }
  });
});
