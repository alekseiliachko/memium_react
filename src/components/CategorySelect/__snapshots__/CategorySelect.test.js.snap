// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`category select all categories render correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiTabs-root WithStyles(ForwardRef(Tabs))-root-2"
  >
    <div
      className="MuiTabs-scroller MuiTabs-fixed"
      onScroll={[Function]}
      style={
        Object {
          "marginBottom": null,
          "overflow": "hidden",
        }
      }
    >
      <div
        className="MuiTabs-flexContainer MuiTabs-centered"
        onKeyDown={[Function]}
        role="tablist"
      >
        <button
          aria-selected={true}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary Mui-selected"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Рекомендации
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Астрология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Биология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Химия
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Айти
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Другое
          </span>
        </button>
      </div>
      <span
        className="PrivateTabIndicator-root-5 PrivateTabIndicator-colorSecondary-7 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-3"
        style={
          Object {
            "left": 0,
            "width": 0,
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`category select all categories render correctly 2`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiTabs-root WithStyles(ForwardRef(Tabs))-root-2"
  >
    <div
      className="MuiTabs-scroller MuiTabs-fixed"
      onScroll={[Function]}
      style={
        Object {
          "marginBottom": null,
          "overflow": "hidden",
        }
      }
    >
      <div
        className="MuiTabs-flexContainer MuiTabs-centered"
        onKeyDown={[Function]}
        role="tablist"
      >
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Рекомендации
          </span>
        </button>
        <button
          aria-selected={true}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary Mui-selected"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Астрология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Биология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Химия
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Айти
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Другое
          </span>
        </button>
      </div>
      <span
        className="PrivateTabIndicator-root-5 PrivateTabIndicator-colorSecondary-7 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-3"
        style={
          Object {
            "left": 0,
            "width": 0,
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`category select all categories render correctly 3`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiTabs-root WithStyles(ForwardRef(Tabs))-root-2"
  >
    <div
      className="MuiTabs-scroller MuiTabs-fixed"
      onScroll={[Function]}
      style={
        Object {
          "marginBottom": null,
          "overflow": "hidden",
        }
      }
    >
      <div
        className="MuiTabs-flexContainer MuiTabs-centered"
        onKeyDown={[Function]}
        role="tablist"
      >
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Рекомендации
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Астрология
          </span>
        </button>
        <button
          aria-selected={true}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary Mui-selected"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Биология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Химия
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Айти
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Другое
          </span>
        </button>
      </div>
      <span
        className="PrivateTabIndicator-root-5 PrivateTabIndicator-colorSecondary-7 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-3"
        style={
          Object {
            "left": 0,
            "width": 0,
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`category select all categories render correctly 4`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiTabs-root WithStyles(ForwardRef(Tabs))-root-2"
  >
    <div
      className="MuiTabs-scroller MuiTabs-fixed"
      onScroll={[Function]}
      style={
        Object {
          "marginBottom": null,
          "overflow": "hidden",
        }
      }
    >
      <div
        className="MuiTabs-flexContainer MuiTabs-centered"
        onKeyDown={[Function]}
        role="tablist"
      >
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Рекомендации
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Астрология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Биология
          </span>
        </button>
        <button
          aria-selected={true}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary Mui-selected"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Химия
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Айти
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Другое
          </span>
        </button>
      </div>
      <span
        className="PrivateTabIndicator-root-5 PrivateTabIndicator-colorSecondary-7 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-3"
        style={
          Object {
            "left": 0,
            "width": 0,
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`category select all categories render correctly 5`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiTabs-root WithStyles(ForwardRef(Tabs))-root-2"
  >
    <div
      className="MuiTabs-scroller MuiTabs-fixed"
      onScroll={[Function]}
      style={
        Object {
          "marginBottom": null,
          "overflow": "hidden",
        }
      }
    >
      <div
        className="MuiTabs-flexContainer MuiTabs-centered"
        onKeyDown={[Function]}
        role="tablist"
      >
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Рекомендации
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Астрология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Биология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Химия
          </span>
        </button>
        <button
          aria-selected={true}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary Mui-selected"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Айти
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Другое
          </span>
        </button>
      </div>
      <span
        className="PrivateTabIndicator-root-5 PrivateTabIndicator-colorSecondary-7 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-3"
        style={
          Object {
            "left": 0,
            "width": 0,
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`category select all categories render correctly 6`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiTabs-root WithStyles(ForwardRef(Tabs))-root-2"
  >
    <div
      className="MuiTabs-scroller MuiTabs-fixed"
      onScroll={[Function]}
      style={
        Object {
          "marginBottom": null,
          "overflow": "hidden",
        }
      }
    >
      <div
        className="MuiTabs-flexContainer MuiTabs-centered"
        onKeyDown={[Function]}
        role="tablist"
      >
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Рекомендации
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Астрология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Биология
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Химия
          </span>
        </button>
        <button
          aria-selected={false}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Айти
          </span>
        </button>
        <button
          aria-selected={true}
          className="MuiButtonBase-root MuiTab-root Component-root-4 MuiTab-textColorPrimary Mui-selected"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="tab"
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiTab-wrapper"
          >
            Другое
          </span>
        </button>
      </div>
      <span
        className="PrivateTabIndicator-root-5 PrivateTabIndicator-colorSecondary-7 MuiTabs-indicator WithStyles(ForwardRef(Tabs))-indicator-3"
        style={
          Object {
            "left": 0,
            "width": 0,
          }
        }
      />
    </div>
  </div>
</div>
`;
